integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/util/IpvCoreStubUtil.java

```diff
@@ -134,4 +134,51 @@ public static HttpResponse<String> sendAuthorizationRequest(
 
         return sendHttpRequest(request);
     }
+
+    public static HttpResponse<String> sendAccessTokenRequest(
+            String devAccessTokenUri, String privateKeyJWT)
```

Comment:
Change this method to receive a String `authorizationCode` rather than `privateKeyJWT`
This class should deal with the creation of the privateKeyJWT